pkg/printers/internalversion/printers.go

```diff
@@ -2603,6 +2610,14 @@ func printPriorityLevelConfigurationList(list *flowcontrol.PriorityLevelConfigur
 	return rows, nil
 }
 
+func printScale(obj *autoscaling.Scale, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+	row.Cells = append(row.Cells, obj.Name, obj.Status.Replicas, obj.Spec.Replicas)
```

Comment:
nit: to follow the deployment format, would it be better to display desiredReplicas first and then availableReplicas? Wdyt?